astroquery/mast/core.py

```diff
@@ -1270,6 +1273,29 @@ def _download_curl_script(self, products, out_dir):
                           "URL": [url]})
         return manifest
 
+    def enable_s3_hst_dataset(self):
+        """
+        Attempts to enable downloading HST public files from S3 instead of MAST
+        """
+        try:
+            global boto3
+            import boto3
+        except ImportError:
+            print("Unable to import boto3.  Using the S3 hst dataset can not be enabled")
+            return
```

Comment:
I'd rather see this just `raise ImportError("Unable to import boto3.  Using the S3 hst dataset can not be enabled")` - users should get exceptions if there's a failure, not just printed statements.